app/src/common/shared/com/igalia/wolvic/ui/views/UITextButton.java

```diff
@@ -12,11 +12,11 @@
 import android.util.AttributeSet;
 
 import androidx.annotation.IdRes;
-import androidx.appcompat.widget.AppCompatButton;
 
+import com.google.android.material.button.MaterialButton;
 import com.igalia.wolvic.R;
 
-public class UITextButton extends AppCompatButton implements CustomUIButton {
+public class UITextButton extends MaterialButton implements CustomUIButton {
```

Comment:
This is what we need! I was getting tons of errors related to this when porting a couple of new architectures we're going to support.